changelogs/fragments/705-pg_hba-invalid-and-parsing.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - "postgresql_pg_hba - fixes #705 by preventing invalid strings to be written"
+  - "postgresql_pg_hba - improves parsing of quoted strings and escaped newlines"
```

Comment:
```suggestion
  - "postgresql_pg_hba - fixes #705 by preventing invalid strings to be written (https://github.com/ansible-collections/community.postgresql/pull/761)"
  - "postgresql_pg_hba - improves parsing of quoted strings and escaped newlines (https://github.com/ansible-collections/community.postgresql/pull/761)"
```

Reply:
@toydarian next time you can just click "Commit suggestion" button so it'd count as our contributions as well:)
Anyway, approved, thanks:)